x-pack/legacy/plugins/infra/common/ecs_allowed_list.ts

```diff
@@ -61,7 +61,10 @@ export const getAllowedListForPrefix = (prefix: string) => {
       if (prefix === 'aws.s3_daily_storage') {
         return [...defaultAllowedList, ...AWS_S3_ALLOWED_LIST];
       }
+      if (prefix === 'aws.metrics') {
+        return [...defaultAllowedList, 'aws.cloudwatch'];
```

Comment:
should the `'aws.cloudwatch'` be in a constant like in the other cases?

Reply:
I will change it to that for consistency sake.